tools/osv-scanner/internal/output/purl_to_package.go

```diff
@@ -0,0 +1,18 @@
+package output
+
+import (
+	"github.com/package-url/packageurl-go"
+)
+
+func PURLToPackage(purl string) (Package, error) {
+	parsedPURL, err := packageurl.FromString(purl)
+	if err != nil {
+		// log.Printf("Failed to parse purl: %s, with error: %s", purl, err)
+		return Package{}, err
+	}
+	return Package{
+		Name:      parsedPURL.Name,
+		Ecosystem: parsedPURL.Type, // TODO: Might want some mapping here to properly cased ecosystems
```

Comment:
Missed this earlier: yeah we will need to fix this. See https://github.com/google/osv.dev/blob/2b01f0db2a37b8628db18a94cdfeaa2eb5e0735c/osv/purl_helpers.py#L18 for some mappings.